packages/amplication-server/src/core/auth/auth.service.ts

```diff
@@ -103,41 +101,40 @@ export class AuthService {
       payload.password
     );
 
-    try {
-      const account = await this.accountService.createAccount({
-        data: {
-          email: payload.email,
-          firstName: payload.firstName,
-          lastName: payload.lastName,
-          password: hashedPassword
-          //role: 'USER'
-        }
-      });
+    const account = await this.accountService.createAccount({
+      data: {
+        email: payload.email,
+        firstName: payload.firstName,
+        lastName: payload.lastName,
+        password: hashedPassword
+      }
+    });
+    const user = await this.bootstrapUser(account, payload.workspaceName);
 
-      const workspace = await this.createWorkspace(
-        payload.workspaceName,
-        account
-      );
+    return this.prepareToken(user);
+  }
 
-      await this.prismaService.project.create({
+  private async bootstrapUser(
+    account: Account,
+    workspaceName: string
+  ): Promise<AuthUser> {
+    const workspace = await this.createWorkspace(workspaceName, account);
+
+    const [user] = workspace.users;
+
+    await this.accountService.setCurrentUser(account.id, user.id);
+
+    await this.projectService.createProject(
```

Comment:
```suggestion
    const workspace = await this.createWorkspace(workspaceName, account);
    const [user] = workspace.users;
    
    await this.accountService.setCurrentUser(account.id, user.id);
    await this.projectService.createProject(
```
We use double line spacing with no meaning here.

I've adjusted it to separate between variable declarations and "await" calls. Then it is meaningful. Otherwise people will assume we just insert double linebreaks after every expression/statement.